src/Microsoft.PowerShell.Commands.Management/commands/management/ControlPanelItemCommand.cs

```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (C) Microsoft.  All rights reserved.
+//    Copyright (c) Microsoft.  All rights reserved.
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."